src/components/tooltip/Tooltip.tsx

```diff
@@ -68,7 +68,7 @@ const ARROW_PADDING = 12;
  *
  * Since the tooltips are appended to the body, they are in the root Stacking Context. If other elements are also there, the z-index will determine which appears on top. By creating a new Stacking Context for your app, it ensures that z-indexed elements will stay within that context, while tooltips will be painted on top, in the parent Stacking Context.
  */
-export const Tooltip = forwardRef<HTMLDivElement, Props>((props, ref) => {
+export const Tooltip = forwardRef<HTMLButtonElement, Props>((props, ref) => {
```

Comment:
Should we actually be more generic with `HTMLElement` ? since we don't know if it's really gonna be a button ?

Reply:
`radixTooltip.Trigger` specifically expects an `HTMLButtonElement` ref

Reply:
I will use `HTMLElement` here and cast to `HTMLButtonElement` when used within `radixTooltip.Trigger`

Reply:
It's weird that radix trigger only accept HTMLButtonElement ref while it's opened to be used with any kind of children